boards/arm/nucleo_f070rb/nucleo_f070rb.dts

```diff
@@ -29,7 +29,8 @@
 		compatible = "gpio-keys";
 		user_button: button {
 			label = "User";
-			gpios = <&gpioc 13 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpioc 13 (GPIO_INT_ACTIVE_LOW |
+					    GPIO_INT_EDGE)>;
```

Comment:
Since `GPIO_INT_ACTIVE_[LOW/HIGH]` depends indeed on HW implementation (button could be wired in active low) it is ok to set it here.

Though `GPIO_INT_EDGE` depends on application requirements and application. One application might require  `INT_EDGE`, the other one `INT_DOUBLE_EDGE`. I'd prefer having those at application level.